src/smc-hub/hub_http_server.coffee

```diff
@@ -268,10 +268,12 @@ exports.init_express_http_server = (opts) ->
             return
         opts.database.get_stats
             cb : (err, stats) ->
+                res.header('Cache-Control', 'private, no-cache, no-store, must-revalidate')
                 if err
                     res.status(500).send("internal error: #{err}")
                 else
-                    res.json(stats)
```

Comment:
It's just for introducing line breaks in the output. I'm really bad at reading json in a single line, that's all :eyes: